packages/core/src/constants.ts

```diff
@@ -1,5 +1,27 @@
 import { GriffelStylesUnsupportedCSSProperties, LookupItem, SequenceHash } from './types';
 
+// ----
+
+// Heads up!
+// These constants are global and will be shared between Griffel instances
+// Any change in them should happen only in a MAJOR version
+
+const __GLOBAL__: any = typeof window === 'undefined' ? global : window;
+const __NAMESPACE_PREFIX__ = '@griffel/';
```

Comment:
> What do you mean?
Well warning that you can have multiple of the same major version on a page... but maybe it's too early since Griffel is still in 1.xx

Reply:
Yeah, I think that it's premature.